content/docs/install/k8s-support/index.md

```diff
@@ -20,8 +20,8 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 
 | Keptn Version /<br>Installation                           | Kubernetes  | AKS                       | EKS                       | GKE           | OpenShift   | K3s         | Minishift               |
 |-----------------------------------------------------------|:-----------:|:-------------------------:|:-------------------------:|:-------------:|:-----------:|:-----------:|:------------------------|
-| **0.17.x** / <br>Control & Execution plane<br>*see: (1)*   | 1.22 - 1.17 | 1.22 - 1.19 | 1.22 - 1.19 | 1.22 - 1.19   | 4, 3.11     | 1.22 - 1.19 | 1.34.2<br>(K8s: 1.11)   |
-| **0.17.x** / <br>Control plane                             | 1.22 - 1.17 | 1.22 - 1.19 | 1.22 - 1.19 | 1.22 - 1.19   | 4, 3.11     | 1.22 - 1.19 | 1.34.2<br>(K8s: 1.11)   |
+| **0.17.x** / <br>Control & Execution plane<br>*see: (1)*   | 1.24 - 1.17 | 1.24 - 1.19 | 1.24 - 1.19 | 1.24 - 1.19   | 4, 3.11     | 1.24 - 1.19 | 1.34.2<br>(K8s: 1.11)   |
+| **0.17.x** / <br>Control plane                             | 1.24 - 1.17 | 1.24 - 1.19 | 1.24 - 1.19 | 1.24 - 1.19   | 4, 3.11     | 1.24 - 1.19 | 1.34.2<br>(K8s: 1.11)   |
```

Comment:
Do we need to also include lines for 0.18.x and 0.19.x, or mark these lines as applying to all three releases?